lib/blacklight.rb

```diff
@@ -70,6 +70,16 @@ def self.connection_config=(value)
     Blacklight::RuntimeRegistry.connection_config = value
   end
 
+  def defaults_version
```

Comment:
What calls this method?

Reply:
Nothing yet. Just getting it out there so we can start opting in to some of the new behavior without needing a major release.